src/aliases/store/migrations/001_add_aliases.up.sql

```diff
@@ -0,0 +1,18 @@
+begin;
+
+create type alias_kind as enum (
```

Comment:
I find this weird... How can a type be 2 things at the same time? An array or a string. This breaks good practices.

Reply:
This is removed entirely. We'll try the idea of just storing a JSON array of string for the keys.